### DISCUSSION TITLE
Welcome to mlbapi Discussions!

### DISCUSSION THREAD

**Discussion by trevor-viljoen** [trevor-viljoen/mlbapi#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mpellet771** [comment]
Hi,
  I am a statistician focused in the nonprofit sector and I love analyzing sports, especially baseball.
